<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VignetteForm</class>
 <widget class="QWidget" name="VignetteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>211</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ptChoice" name="MaskType"/>
     </item>
     <item>
      <widget class="ptChoice" name="Shape"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ptInput" name="Strength"/>
   </item>
   <item>
    <widget class="ptInput" name="InnerRadius"/>
   </item>
   <item>
    <widget class="ptInput" name="OuterRadius"/>
   </item>
   <item>
    <widget class="ptInput" name="Roundness"/>
   </item>
   <item>
    <widget class="ptInput" name="CenterX"/>
   </item>
   <item>
    <widget class="ptInput" name="CenterY"/>
   </item>
   <item>
    <widget class="ptInput" name="Softness"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
  <customwidget>
   <class>ptChoice</class>
   <extends>QComboBox</extends>
   <header>ptChoice.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
